src/provsalt/lightningdeath/LightningListener.php

```diff
@@ -41,6 +43,9 @@ public function Lightning(Player $player) :void{
 			$light->pitch = $player->getPitch();
 			$light->position = new Vector3($player->getX(), $player->getY(), $player->getZ());
 			Server::getInstance()->broadcastPacket($player->getLevel()->getPlayers(), $light);
+			$block = $player->getLevel()->getBlock($player->getPosition()->floor()->down());
+			$particle = new DestroyBlockParticle(new Vector3($player->getX(), $player->getY(), $player->getZ()), new Block($block->getId()));
```

Comment:
why do you have to create a new Block instance where $block is an instance of a Block

Reply:
oh-